package.json

```diff
@@ -2024,144 +2024,158 @@
           "fontCharacter": "\\f101"
         }
       },
+      "confluent-code": {
+        "description": "confluent-code icon",
+        "default": {
+          "fontPath": "./resources/dist/icons.woff2",
+          "fontCharacter": "\\f102"
+        }
+      },
       "confluent-environment": {
         "description": "confluent-environment icon",
         "default": {
           "fontPath": "./resources/dist/icons.woff2",
-          "fontCharacter": "\\f102"
+          "fontCharacter": "\\f103"
```

Comment:
these seem to be positional, so `gulp icongen` changes them when new icons are added